docs/android.md

```diff
@@ -1,66 +1,40 @@
 ## Integrating Unity as a library into standard Android app
-This document explains how to include Unity as a Library into standard Android application through Activity. You can read more about [Unity as a Library](https://docs.unity3d.com/2019.3/Documentation/Manual/UnityasaLibrary.html).
+This document explains how to include Unity as a Library into standard Android application through Activity. You can read more about [Unity as a Library](https://docs.unity3d.com/6000.0/Documentation/Manual/UnityasaLibrary.html).
 
 **Requirements:**
-- Android Studio Hedgehog (2023.1.1) or later
-- Unity version 2023.1.7f1, 2023.2.0b3 or later
+- Android Studio Iguana (2023.2.1) or later
+- Unity version 6000.0.0b16 or later
 
-[Note] For Unity versions from 2019.3.0b4 to 2022.2.0a17 use [19LTS-21LTS branch](https://github.com/Unity-Technologies/uaal-example/tree/uaal-example/19LTS-21LTS). For Unity versions from 2022.2.0a17 to 2023.1.0a16 use [22LTS branch](https://github.com/Unity-Technologies/uaal-example/tree/uaal-example/22LTS).
+  **Note**
```

Comment:
The formatting is broken a bit, check https://github.com/Unity-Technologies/uaal-example/blob/578fd3b27a37765b8450628ecb263416d5755d29/docs/android.md

it should list the items similarly to Requirements section

Reply:
Oh, I thought Note should belong/under Requirements as it was in the past.
You think Note would be the same level as Requirements, correct?